recipe/meta.yaml

```diff
@@ -6,38 +6,46 @@ package:
   version: {{ version }}
 
 source:
-  url: https://lhapdf.hepforge.org/downloads/?f=LHAPDF-{{ version }}.tar.gz
-  sha256: 2443a4b32cc3b0597c8248bd6e25703ace9c91a7a253c5f60b1b5428ef9c869e
+  url: https://gitlab.com/hepcedar/lhapdf/-/archive/lhapdf-{{ version }}/lhapdf-lhapdf-{{ version }}.tar.gz
+  sha256: ace8913781044ad542e378697fcd95a8535d510818bb74a6665f9fd2b132ac0f
```

Comment:
Building from GitLab means that you're starting from the source with no additional changes like the `configure` script being already generated (as the hepforge releases do), so you get a more uniform experience of building across all platforms. This is also going to require that all platforms install the autotools requirements as `build` requirements.